drivers/tty/serdev/pibridge.c

```diff
@@ -259,6 +259,9 @@ int pibridge_req_gate_tmt(u8 dst, u16 cmd, u8 *snd_buf, u16 snd_len,
 				dst, cmd, snd_len);
 		return -EIO;
 	}
+	/* Do not wait for a response in case of a broadcast address */
+	if (dst == 0xff)
+		return 0;
```

Comment:
Interesting. Does the existing code in piControl do the same?

Reply:
Yes. Would be better to use a constant/makro for the "0xff", though